src/panels/config/voice-assistants/assist-pipeline-detail/assist-pipeline-detail-conversation.ts

```diff
@@ -31,18 +35,33 @@ export class AssistPipelineDetailConversation extends LitElement {
                 },
               },
             },
-            supportedLanguages !== "*" && supportedLanguages?.length
-              ? {
-                  name: "conversation_language",
-                  required: true,
-                  selector: {
-                    language: { languages: supportedLanguages, no_sort: true },
-                  },
-                }
-              : { name: "", type: "constant" },
-          ] as const,
+          ],
         },
-      ] as const
+      ];
+
+      if (supportedLanguages !== "*" && supportedLanguages?.length) {
+        fields[0].schema.push({
+          name: "conversation_language",
+          required: true,
+          selector: {
+            language: { languages: supportedLanguages, no_sort: true },
+          },
+        });
+      }
+
+      if (engine !== "conversation.home_assistant") {
+        fields.push({
+          name: "conversation_process_local",
+          required: true,
+          default: true,
```

Comment:
Does `default: true` actually do anything? It doesn't seem to enable the option by default and that wouldn't work for existing pipelines anyway